artemis-journal/src/main/java/org/apache/activemq/artemis/core/journal/impl/JournalImpl.java

```diff
@@ -2858,16 +2858,19 @@ public synchronized void start() {
    }
 
    @Override
-   public synchronized void stop() throws Exception {
+   public synchronized void stop() {
       if (state == JournalState.STOPPED) {
          return;
       }
 
-      flush();
+      try {
+         flush();
+      } catch (Exception e) {
+         logger.warn(e.getMessage(), e);
```

Comment:
Warning messages like this should use a logger with a corresponding code.

Reply:
hmmmm... I'm not really sure .. I don't see much point for this one... logging the ongoing exception like the caller would throw should be enough on this one

Reply:
fixed though